cmd/ampmake/build

```diff
@@ -7,11 +7,14 @@ DEST=$GOPATH/src/$REPO/bin
 TARGETS=(
     darwin amd64
     linux amd64
+    windows amd64
 )
 
 for ((i=0; i<"${#TARGETS[@]}"; $((i=$i+2)) )); do
   GOOS="${TARGETS[i]}"
   GOARCH="${TARGETS[i+1]}"
-  echo $DEST/$GOOS/$GOARCH/ampmake
-  GOOS=$GOOS GOARCH=$GOARCH go build -o $DEST/$GOOS/$GOARCH/ampmake $PKG || exit $?
+  OUTPUT=$DEST/$GOOS/$GOARCH/ampmake
+  if [[ $GOOS = "windows" ]]; then OUTPUT=$OUTPUT".exe"; fi
```

Comment:
Unfortunately, this doesn't do any good right now since `ampmake` can't run on Windows because it mounts the Docker socket. We won't be able to support hacking on amp itself just yet. The only thing we can support right now is running amp on Windows for deploying stacks to a hosted environment, but not creating a local DinD-based swarm cluster and not running `ampmake` to work on amp.